public/app/features/plugins/admin/helpers.ts

```diff
@@ -126,8 +126,8 @@ export function mapRemoteToCatalog(plugin: RemotePlugin, error?: PluginError): C
     id,
     info: {
       logos: {
-        small: `https://grafana.com/api/plugins/${id}/versions/${version}/logos/small`,
-        large: `https://grafana.com/api/plugins/${id}/versions/${version}/logos/large`,
+        small: `${config.appSubUrl}/api/gnet/plugins/${id}/versions/${version}/logos/small`,
```

Comment:
Is this going via the grafana server gnet proxy? Is so do we need `config.appSubUrl`?

Reply:
it does, and yes we do if we run from a subfolder